public/install.php

```diff
@@ -66,6 +66,14 @@
 			<?php endif ?>
 		</tr>
 		<tr>
+			<th>Composer Installed</th>
+			<?php if (is_dir($vendor_path) AND is_file($vendor_path.'autoload.php')): ?>
+				<td class="pass"><?php echo $vendor_path ?></td>
```

Comment:
Then again, I would raise the consistency point: if we change it here, we have to change it everywhere and we are still using the traditional `<?php` tag.